src/components/shared/TreeView.css

```diff
@@ -0,0 +1,229 @@
+  position: absolute;
+  top: 3px;
+  right: 0;
+  bottom: 3px;
+  width: 1px;
+  background: #e5e5e5;
+  content: '';
+}
+.treeViewHeaderColumn.total,
+.treeViewHeaderColumn.self {
+  text-align: right;
+}
+.treeViewRowColumn.treeViewFixedColumn {
+  overflow: hidden;
+  box-sizing: border-box;
+  border-right: 1px solid var(--grey-30);
+  text-overflow: ellipsis;
+}
+.treeViewFixedColumn.total {
+  left: 0;
+  width: 70px;
+}
+.treeViewFixedColumn.totalPercent {
+  left: 70px;
+  width: 50px;
+  border-right: none;
+}
+.treeViewFixedColumn.self {
+  left: 120px;
+  width: 70px;
+}
+.treeViewHeaderColumn.total {
+  width: 120px;
+}
+.treeViewFixedColumn.icon {
+  left: 190px;
+  display: flex;
+  width: 19px;
+  flex-flow: column nowrap;
+  align-items: center;
+}
+.treeViewRowColumn.total,
+.treeViewRowColumn.totalPercent,
+.treeViewRowColumn.self,
+.treeViewRowColumn.timestamp {
+  padding-right: 5px;
+  text-align: right;
+}
+.treeViewRowColumn.type {
+  padding-left: 5px;
+  text-align: left;
+}
+
+.treeRowIndentSpacer {
+  flex-shrink: 0;
+}
+
+.treeViewAppendageColumn {
+  margin-left: 10px;
+  opacity: 0.6;
+}
+
+.treeViewRow.even {
+  background-color: #fff;
+}
+.treeViewRow.odd {
+  background-color: #f5f5f5;
+}
+
+/* Note that `isRightClicked` is present before `isSelected` so that it can be overriden. */
+.treeViewRow.isRightClicked {
+  background-color: var(--blue-50-a30);
+}
+.treeViewRow.isSelected {
+  background-color: var(--grey-30);
+  color: black;
+}
+.treeViewRow.dim > .treeViewMainColumn {
+  opacity: 0.7;
+}
+.treeViewBody:focus .treeViewRow.isSelected {
+  background-color: highlight;
+  color: highlighttext;
+}
+.treeViewBody:focus .treeViewRow.isSelected a {
+  color: inherit;
+}
+
+.treeViewHighlighting {
+  padding: 1px 0;
+
+  /* This negative margin enlarges the background to the top, so that it fully
+   * covers the underlying background. There's an underlying background when the
+   * line is selected. */
+  margin: -1px 0;
+  background: #cbe8fe;
+  border-radius: 2px;
+  box-shadow: 0 0 0 0.5px rgba(0, 0, 0, 0.05), 0 1px 1px rgba(0, 0, 0, 0.3);
+  color: #000;
+}
+
+.treeRowToggleButton {
+  width: 0;
+  height: 0;
+  border-top: 5px solid transparent;
+  border-right: 4px solid transparent;
+  border-bottom: 5px solid transparent;
+  border-left: 8px solid;
+  margin-left: 8px;
+  color: #888;
+}
+
+.treeRowToggleButton.expanded {
+  border-top: 8px solid;
+  border-right: 5px solid transparent;
+  border-bottom: 4px solid transparent;
+  border-left: 5px solid transparent;
+  margin-top: 5px;
+  margin-right: 5px;
+  margin-left: 5px;
+}
+
+.treeRowToggleButton:active:hover {
+  color: var(--grey-60);
+}
+
+.treeViewBody:focus .treeViewRow.isSelected > .treeRowToggleButton {
+  color: #fff;
+}
+
+.treeViewBody:focus
+  .treeViewRow.isSelected
+  > .treeRowToggleButton:active:hover {
+  color: rgba(255, 255, 255, 0.7);
+}
+
+.treeRowToggleButton.leaf {
+  visibility: hidden;
+}
+
+.treeAndSidebarWrapper {
```

Comment:
This class belongs to `ProfileCallTreeView` component which is outside of `TreeView`. Could you move this back?

Reply:
My bad :), I moved it out to it's original location, 'style.css'.